packages/runtime/test/lib/testUtils.ts

```diff
@@ -294,9 +299,8 @@ export async function sendMessageToMultipleRecipients(transportServices: Transpo
         },
         attachments
     });
-    expect(response).toBeSuccessful();
```

Comment:
Why was this removed?

Reply:
Otherwise I cannot use it for both cases the sucessful one and the one where it throws an error.